website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,767 @@
+				a Prometheus exporter) to Datadog. Previously these metrics were dropped at the sink.
+				"""
+			pr_numbers: [12436]
+		},
+		{
+			type: "fix"
+			scopes: ["internal_logs source"]
+			description: """
+				The `internal_logs` source occasionally missed some events generated early in Vector's start-up, before
+				the component was initialized. This was remedied so that the `internal_logs` source more reliably
+				captures start-up events.
+				"""
+			pr_numbers: [12411]
+		},
+		{
+			type: "feat"
+			scopes: ["gcp_pubsub source", "sources"]
+			description: """
+				A new `gcp_pubsub` source was added for consuming events from [GCP PubSub](https://cloud.google.com/pubsub).
+				"""
+			pr_numbers: [12057]
+		},
+		{
+			type: "feat"
+			scopes: ["websocket sink", "sinks"]
+			description: """
+				A new `websocket` sink was added for sending events to a remote websocket listener.
+				"""
+			pr_numbers: [9632]
+			contributors: ["@zshell31"]
+		},
+		{
+			type: "fix"
+			scopes: ["vrl"]
+			description: """
+				The `parse_ruby_hash` VRL function can now parse hashes that contain a symbol as the value, such as `{ "key" => :foo }`.
+				"""
+			pr_numbers: [12514]
+		},
+		{
+			type: "enhancement"
+			scopes: ["releasing"]
+			description: """
+				The RPM package now adds the created `vector` user to the `systemd-journal-remote` group to be able to
+				consume journald events from a remote system. This matches the Debian package.
+				"""
+			pr_numbers: [12563]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now allows configuration of `extra_namespace_label_selector` which Vector
+				will to use select the pods to capture the logs of, if set, based on labels attached to the pod
+				namespace. This is similar to the `extra_label_selector` option which applies to pod labels.
+				"""
+			contributors: ["@anapsix"]
+			pr_numbers: [12438]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now reads events in order whenever a pod log file rotates. Previously Vector could start reading the new file before it finished processing the previous one, resulting in the logs being out-of-order.
+				"""
+			pr_numbers: [12330]
+			contributors: ["@sillent"]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl"]
+			description: """
+				The `parse_json` function now takes an optional `max_depth` parameter to control how far it will recurse when deserializing the event. Once the depth limit is hit, the remainder of the fields is left as raw JSON in the deserialized event.
+
+				For example:
+
+				```coffeescript
+				parse_json!("{\"1\": {\"2\": {\"3\": {\"4\": {\"5\": {\"6\": \"finish\"}}}}}}", max_depth: 5)
+				```
+
+				Yields:
+
+				```json
+				{ "1": { "2": { "3": { "4": { "5": "{\"6\": \"finish\"}" } } } } }
+				```
+
+				The default remains no max depth limit.
```

Comment:
I wonder if it makes sense to have a special "VRL changes" section, to specifically call out function additions/changes (and less common, syntax changes)?

Reply:
Yeah, I thought about that too. I'll leave it as-is for now, but I could see us coming up with better groupings in the future rather than listing each change independently.